### PR TITLE
Shades can move in space

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Shades can move in space.
rscadd: Artificers can now heal shades.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/shade.dm [hunk 1]
```diff
@@ -26,21 +26,32 @@
...
-	new /obj/item/weapon/ectoplasm (src.loc)
-	visible_message("<span class='warning'>[src] lets out a contented sigh as their form unwinds.</span>")
-	ghostize()
-	qdel(src)
-	return
+	deathmessage = "<span class='warning'>[src] lets out a contented sigh as their form unwinds.</span>"
+	..()
 
+/mob/living/simple_animal/shade/Process_Spacemove(movement_dir = 0)
+	return 1
```
Comment (coiax) [inline code/modules/mob/living/simple_animal/shade.dm]: shouldn't this be TRUE

Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/shade.dm]: All the other ones aren't but probably, yes. It's one of the things under "someone else's problem"

Comment (coiax) [inline code/modules/mob/living/simple_animal/shade.dm]: In most languages TRUE is just a special case of the number 1. Is that not true (heh) in byond?

Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/shade.dm]: It's TRUE in byond, yes. It's also more or less *someone else's problem.*

Comment (pudl) [inline code/modules/mob/living/simple_animal/shade.dm]: fix this shit

Comment (optimumtact) [inline code/modules/mob/living/simple_animal/shade.dm]: >LINE YOU TOUCHED
>NOT MY PROBLEM LMAO

no it doesn't work like that Joan, make it true

Comment (PKPenguin321) [inline code/modules/mob/living/simple_animal/shade.dm]: @ChangelingRain here i did it for you just hit merge: https://github.com/ChangelingRain/-tg-station/pull/7

**Pull request merged**